compiler-rt/lib/asan/asan_globals.cpp

```diff
@@ -35,8 +36,11 @@ struct ListOfGlobals {
   ListOfGlobals *next;
 };
 
+typedef DenseMap<uptr, ListOfGlobals *> MapOfGlobals;
```

Comment:
typedef DenseMap<uptr, IntrusiveList<ListOfGlobals>> MapOfGlobals;

Reply:
Replaced here and in the main list for consistency https://github.com/llvm/llvm-project/pull/100923/commits/151257d3d68e932b1b13f45f3b204a2400637ce8

Reply:
It would be nice to have such small refactoring out of real patches.
Assuming we are in different time zones, to avoid unnecessary iterations, I will extract land it separately.

Reply:
Thanks @vitalybuka, appreciate it! 

P.S. If there's a guide on how to do "stacked" PRs on Github or otherwise what's the blessed workflow for stacks of changes that'd be very helpful (not much about this in https://llvm.org/docs/GitHub.html). I'd do a stack of diffs in Phabricator but here it's different.

Reply:
Thanks for the pointers and the explanation @vitalybuka!